aspnetcore/mvc/models/file-uploads/samples/3.x/SampleApp/appsettings.json

```diff
@@ -7,6 +7,6 @@
     }
   },
   "AllowedHosts": "*",
-  "StoredFilesPath": "c:\\files",
-  "FileSizeLimit": 2097152
+  "StoredFilesPath": "D:\\tmp\\LargeFileUpload",
+  "FileSizeLimit": 6442450944
```

Comment:
I calculated this in bytes (6 GiB) but it might actually be in MiB (i.e. 6 TiB).  🤷